am-lib/src/main/java/uk/gov/hmcts/reform/amlib/repositories/AccessManagementRepository.java

```diff
@@ -32,7 +32,8 @@
         + "and access_management.service_name = :serviceName "
         + "and access_management.resource_type = :resourceType "
         + "and access_management.resource_name = :resourceName "
-        + "and access_management.attribute = :attributeAsString")
+        + "and access_management.attribute = :attributeAsString "
+        + "or access_management.attribute ilike concat(:attributeAsString, '/','%')")
```

Comment:
Either use `like` or have tests for behaviour given by `ilike`.

At this point regular like would be sufficient to be consistent with rest of the library. We need to look into case matching at some point through.

Reply:
👍